pkg/catalog/catalog.go

```diff
@@ -1772,7 +1772,13 @@ type UncommittedParquetObject struct {
 	CreationDate    int64  `parquet:"name=creation_date, type=INT64, convertedtype=INT_64"`
 }
 
-func (c *Catalog) writeUncommittedLocal(ctx context.Context, repository *graveler.RepositoryRecord, w *UncommittedWriter, pw *writer.ParquetWriter, mark *GCUncommittedMark, runID string) (*GCUncommittedMark, bool, error) {
+func (c *Catalog) writeUncommittedLocal(ctx context.Context, repository *graveler.RepositoryRecord, w *UncommittedWriter, mark *GCUncommittedMark, runID string) (*GCUncommittedMark, bool, error) {
```

Comment:
Since we don't write the linked addresses anymore - I think we can revert the changes in writeUncommittedLocal

Reply:
done